docs: langflow 1.5 auto-login security doc (#9057)
langflow-ai/langflow: docs/docs/Configuration/environment-variables.mdx

| <Link id="LANGFLOW_WORKERS"/><span class="env-prefix">LANGFLOW_</span>WORKERS | Integer | `1` | Number of worker processes.<br/>See [`--workers` option](./configuration-cli.mdx#run-workers). |
| <Link id="LANGFLOW_SSL_CERT_FILE"/><span class="env-prefix">LANGFLOW_</span>SSL_CERT_FILE | String | Not set | Path to the SSL certificate file on the local system. |
| <Link id="LANGFLOW_SSL_KEY_FILE"/><span class="env-prefix">LANGFLOW_</span>SSL_KEY_FILE | String | Not set | Path to the SSL key file on the local system. |

> | <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | If set to `true`, skips authentication and allows automatic login as the superuser, regardless of the value of `LANGFLOW_AUTO_LOGIN`. If set to `false`, disables automatic login and enforces authentication, regardless of the value of `LANGFLOW_AUTO_LOGIN`. |

Reviewer: `LANGFLOW_SKIP_AUTH_AUTO_LOGIN` only has an effect if `LANGFLOW_AUTO_LOGIN` is `true`. If `LANGFLOW_AUTO_LOGIN` is `false`, it does nothing.

> | <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | If set to `true`, skips authentication and allows automatic login as the superuser, regardless of the value of `LANGFLOW_AUTO_LOGIN`. If set to `false`, disables automatic login and enforces authentication, regardless of the value of `LANGFLOW_AUTO_LOGIN`. |

Author: If set to true, and `LANGFLOW_AUTO_LOGIN` is set to `true`, skips authentication and allows automatic login as the superuser. If set to false, and `LANGFLOW_AUTO_LOGIN` is set to true, disables automatic login and enforces authentication. Has no effect if `LANGFLOW_AUTO_LOGIN` is false.

> | <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | If set to `true`, skips authentication and allows automatic login as the superuser, regardless of the value of `LANGFLOW_AUTO_LOGIN`. If set to `false`, disables automatic login and enforces authentication, regardless of the value of `LANGFLOW_AUTO_LOGIN`. |

Author: ```suggestion
| <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | disables automatic login and enforces authentication, regardless of the value of `LANGFLOW_AUTO_LOGIN`. |
```

> | <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | If set to `true`, skips authentication and allows automatic login as the superuser, regardless of the value of `LANGFLOW_AUTO_LOGIN`. If set to `false`, disables automatic login and enforces authentication, regardless of the value of `LANGFLOW_AUTO_LOGIN`. |

Author: ```suggestion
| <Link id="LANGFLOW_SKIP_AUTH_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>SKIP_AUTH_AUTO_LOGIN | Boolean | `true` | If set to `true`, and `LANGFLOW_AUTO_LOGIN` is set to `true`, skips authentication and allows automatic login as the superuser. If set to `false`, and `LANGFLOW_AUTO_LOGIN` is set to `true`, disables automatic login and enforces authentication. If `LANGFLOW_AUTO_LOGIN ` is `false`, has no effect. |
```
